src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -568,6 +567,18 @@ public static List<ClusterMetadata> tcmLoadRange(long min, long max)
         return afterLoad;
     }
 
+    private static List<ClusterMetadata> reconstruct(long min, long max)
+    {
+        Epoch start = Epoch.create(min);
+        Epoch end = Epoch.create(max);
+        Retry.Deadline deadline = Retry.Deadline.wrap(new Retry.ExponentialBackoff(42,
```

Comment:
Is there any reason we're not using TCM configurable here? I'll also be OK if we let it be configured it some other way.

Reply:
spoke in slack, changed to use max retry and default backoff for the first 2 params; added a new max backoff for the 3rd param as a config.  

new defaults match `10, 50ms, 1s`